GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg

```diff
@@ -0,0 +1,780 @@
+}
+
+
+//	Starliner CM
+@PART[BCS_Centauri_CrewCapsule]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.6
+	@title = CST-100 Starliner Command Module
+	@manufacturer = #roMfrBoeing
+	@description = The Command Module of the CST-100 Starliner. Produced by Boeing for Crew Transportation to the International Space Station as part of NASA's Commercial Crew Program. Designed to be reusable for up to 10 times.
+	@mass = 4.35
+	%skinMassPerArea = 15.99
+	%skinTempTag = HRSI
+	%internalTempTag = Instruments
+	%skinInsulationTag = True
+	!RESOURCE,* {}
+	@MODULE[ModuleCommand]
+	{
+		@RESOURCE[ElectricCharge]
+		{
+			@rate = 1.6
+		}
+	}
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[ReentryCoM]] {}
+	MODULE
+	{
+		name = AdjustableCoMShifter
+		DescentModeCoM = 0.0, 0.0, 0.158
+	}
+
+	//	CM RCS:
+	//	12x MR-104J RCS Engines
+	//	Fuel: Hydrazine
+	//	Thruster Power: 100lbf (445N)
+	//	SL ISP: 215
+	//	Vac ISP: 223
+	//	Mass: 6.44kg
+
+	@MODULE[ModuleRCSFX]
+	{
+		!resourceName = NULL
+		!atmosphereCurve {}
+		@thrusterPower = 0.445
+		PROPELLANT
+		{
+			name = Hydrazine
+			ratio = 1.0
+		}
+		PROPELLANT
+		{
+			name = Helium
+			ratio = 10.0
+			ignoreForIsp = True
+		}
+		atmosphereCurve
+		{
+			key = 0 223
+			key = 1 215
+		}
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 180
+		basemass = -1
+		//	Batteries 1.5 kWh.
+		TANK
+		{
+			name = ElectricCharge
+			amount = 5400
+			maxAmount = 5400
+		}
+		//	ACS propellant mass 150 Kg.
+		TANK
+		{
+			name = Hydrazine
+			amount = 150
+			maxAmount = 150
+		}
+		//	ACS pressurization gas mass ~0.27 Kg.
+		TANK
+		{
+			name = Helium
+			amount = 1500
+			maxAmount = 1500
+		}
+	}
+	!MODULE[ModuleDataTransmitter],* {}
+	MODULE
+	{
+		name = ModuleRealAntenna
+		referenceGain = 1.5
+		RFBand = S		//Default to S-band
+	}
+	MODULE
+	{
+		name = ModuleRealAntenna
+		antennaDiameter = 0.8	//Same as Orion because I cannot find any data
+		RFBand = K	//Default to K-band
+	}
+}
+@PART[BCS_Centauri_CrewCapsule]:NEEDS[RealFuels]:AFTER[ROCapsules]
+{
+	@description ^= :$: Supports a crew of 7 for 2 days.:
+	@mass -= 0.07
+	@MODULE[ModuleFuelTanks]
+	{
+		//	7 Crew for 2 Days, Tanks filled for 4 people for ~4 Days
+		@volume = 460
+		TANK
+		{
+			name = Food
+			amount = 100
+			maxAmount = 100
+		}
+		TANK
+		{
+			name = Water
+			amount = 67
+			maxAmount = 67
+		}
+		TANK
+		{
+			name = Oxygen
+			amount = 10000
+			maxAmount = 10000
+		}
+		TANK
+		{
+			name = Nitrogen
+			amount = 200
+			maxAmount = 200
+		}
+		TANK
+		{
+			name = Waste
+			amount = 0
+			maxAmount = 9.32
+		}
+		TANK
+		{
+			name = WasteWater
+			amount = 0
+			maxAmount = 34.47
+		}
+	}
+}
+@PART[BCS_Centauri_CrewCapsule]:NEEDS[ProfileRealismOverhaul]:AFTER[RealismOverhaul]
```

Comment:
This should be in the kerbalism profile, not RO.

Reply:
Moved to ROKerbalism